src/chrome/content/rules/bplaced.xml

```diff
@@ -0,0 +1,23 @@
+<!--
+
+	Non-functional subdomain:
+		- forum
+
+-->
+
+<ruleset name="bplaced">
+
+	<target host=    "bplaced.net" />
+	<target host="www.bplaced.net" />
+
+  	<securecookie host="^www\.bplaced\.net$" name=".*" />
+
```

Comment:
Why redirect to www? ^bplaced.net seems to work fine. If there's no problem, please don't preempt the server's redirect. Doing so only gives an adversary on the wire greater confidence that the user is using https-everywhere.